### DISCUSSION TITLE
Create/edit modal label

### DISCUSSION THREAD

**Discussion by matijarezek** [filamentphp/filament#9807] [Help] [answered]
I have a resource for users to create new tickets.

`protected static ?string $label = 'My Tickets';
  protected static ?string $modelLabel = 'My Tickets';`

I have this label set up and i have commented out the edit page so the modal opens up when they want to create a new ticket.

Now, the problem is on the modal the label is "Create My Tickets" and I want it to be "Create new ticket".

How can I accomplish that?

**Comment by leandrocfe** [comment, marked as answer]
add modalHeading in the create action

```php
CreateAction::make()
    ->modalHeading('Create New ticket')
```

**Reply by matijarezek** [reply]
Amazing and simple. I was searching for a "label" with intellisense but I guess heading does make more sense. 

Thank u kind sir!
